.env.local.example

```
@@ -1,3 +1,3 @@
-# Supabase (ブラウザ公開可の値)
-NEXT_PUBLIC_SUPABASE_URL=your-url
-NEXT_PUBLIC_SUPABASE_ANON_KEY=your-anon-key
+NEXT_PUBLIC_SUPABASE_URL=https://rlopopbtdydqchifxla.supabase.co
+NEXT_PUBLIC_SUPABASE_ANON_KEY=sb_publishable_0pac5l4QBih95XXJVWdgAA_JGNB9YUd
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SUPABASE_ANON_KEY key should go before the NEXT_PUBLIC_SUPABASE_URL key

(UnorderedKey)